CONTRIBUTING.md

```diff
@@ -194,40 +215,66 @@ https://github.com/google/google-java-format/blob/master/README.md#intellij-jre-
 3. Update [COMPATIBILITY_MATRIX.md](COMPATIBILITY_MATRIX.md).
+
+#### Procedure used before July 1, 2024
+
 4. Update IRS API version in IrsApplication class and irs-api.yaml
-5. Create pull request from [release preparation branch to main](https://github.com/eclipse-tractusx/item-relationship-service/compare/chore/prepare-release-x.x.x) 
+5. Create pull request
+   from [release preparation branch to main](https://github.com/eclipse-tractusx/item-relationship-service/compare/chore/prepare-release-x.x.x)
 6. Merge this pull request to main.
 7. Create Git tag for the desired release version `git tag x.x.x`
-   (note: the _item-relationship-service_ tag will be created automatically by the GitHub workflow based on the version in the [helm chart changelog](charts/item-relationship-service/CHANGELOG.md)). 
+   (note: the _item-relationship-service_ tag will be created automatically by the GitHub workflow based on the version
+   in the [helm chart changelog](charts/item-relationship-service/CHANGELOG.md)).
 8. Push Git tag to repository `git push origin x.x.x` (this will trigger the GitHub release workflow).
 9. Wait for release workflow to complete.
 10. Merge the pull request that was automatically opened by GitHub actions bot.
-11. Notify about the release in IRS Matrix Chat using the following template: 
-   
-   >   **IRS Release x.x.x**
-   >
-   >   IRS version x.x.x is released. 
-   >
-   >   https://github.com/eclipse-tractusx/item-relationship-service/releases/tag/x.x.x<br>
-   >   https://github.com/eclipse-tractusx/item-relationship-service/releases/tag/item-relationship-service-y.y.y<br>
-   >   **Full Changelog:** https://github.com/eclipse-tractusx/item-relationship-service/compare/w.w.w...x.x.x
 
-   _(replace x.x.x with IRS version to release, y.y.y with IRS helm version to release and w.w.w with previous IRS version)_
+#### Procedure used starting July 1, 2024
+
+Starting July 1, 2024, a new version of the release workflow is used which automates the changes that previously
+had to be done by hand.
+
+4. Execute the workflow named _"Release IRS"_. The workflow takes three inputs. These are:
+    - IRS release version - type: semantic version string
+    - Helm Chart release version - type: semantic version string
+    - Whether to automatically update the Helm Chart changelog with the change
+      _"- Update IRS version to x.x.x"_. This is a checkbox which is ticked by default.
+5. The workflow makes the automated changes and creates a pull request from them right away.
+   Please review the created pull request within 15 minutes. If you fail to do so, or if the
+   pull request gets closed, the workflow will fail.
+6. Once the pull request has been merged, the workflow continues execution. It pushes a tag (which will be
+   the semantic IRS release version number) and creates the release. Subsequent workflows, such
+   as the IRS build workflow, are triggered automatically.
 
```

Comment:
That's not quite the hint I had in mind. Please just stick to the "new" way of creating a release

Reply:
Hint text now changed to "old way" and "new way"